web/src/lib/roadmap/buildUserRoadmap.ts

```diff
@@ -216,7 +216,7 @@ const getLegalStructureAddOns = (profileData: ProfileData): string[] => {
   if (profileData.legalStructureId === "nonprofit") {
     addOns.push("nonprofit");
 
-    const allowNonprofitFormation = allowFormation(profileData.legalStructureId);
+    const allowNonprofitFormation = allowFormation(profileData.legalStructureId, profileData.businessPersona);
```

Comment:
[sand] I don't think we need this or the check on 220. All nonprofits (foreign and domestic) have formation enabled.